template/src/index.tsx

```diff
@@ -23,47 +23,55 @@ import Sidebar from './components/Sidebar';
   const root = document.getElementById('root');
+  render(<LocalhostWarning />, root);
+} else {
+  init((sdk: BaseExtensionSDK) => {
+    const root = document.getElementById('root');
 
-  // All possible locations for your app
-  // Feel free to remove unused locations
-  // Dont forget to delete the file too :)
-  const ComponentLocationSettings = [
-    {
-      location: locations.LOCATION_APP_CONFIG,
-      component: <Config sdk={(sdk as unknown) as AppExtensionSDK} />
-    },
-    {
-      location: locations.LOCATION_ENTRY_FIELD,
-      component: <Field sdk={(sdk as unknown) as FieldExtensionSDK} />
-    },
-    {
-      location: locations.LOCATION_ENTRY_EDITOR,
-      component: <EntryEditor sdk={(sdk as unknown) as EditorExtensionSDK} />
-    },
-    {
-      location: locations.LOCATION_DIALOG,
-      component: <Dialog sdk={(sdk as unknown) as DialogExtensionSDK} />
-    },
-    {
-      location: locations.LOCATION_ENTRY_SIDEBAR,
-      component: <Sidebar sdk={(sdk as unknown) as SidebarExtensionSDK} />
-    },
-    {
-      location: locations.LOCATION_PAGE,
-      component: <Page sdk={(sdk as unknown) as PageExtensionSDK} />
-    }
-  ];
+    // All possible locations for your app
+    // Feel free to remove unused locations
+    // Dont forget to delete the file too :)
+    const ComponentLocationSettings = [
+      {
+        location: locations.LOCATION_APP_CONFIG,
+        component: <Config sdk={(sdk as unknown) as AppExtensionSDK} />
+      },
+      {
+        location: locations.LOCATION_ENTRY_FIELD,
+        component: <Field sdk={(sdk as unknown) as FieldExtensionSDK} />
+      },
+      {
+        location: locations.LOCATION_ENTRY_EDITOR,
+        component: <EntryEditor sdk={(sdk as unknown) as EditorExtensionSDK} />
+      },
+      {
+        location: locations.LOCATION_DIALOG,
+        component: <Dialog sdk={(sdk as unknown) as DialogExtensionSDK} />
+      },
+      {
+        location: locations.LOCATION_ENTRY_SIDEBAR,
+        component: <Sidebar sdk={(sdk as unknown) as SidebarExtensionSDK} />
+      },
+      {
+        location: locations.LOCATION_PAGE,
+        component: <Page sdk={(sdk as unknown) as PageExtensionSDK} />
+      }
+    ];
 
-  // Select a component depending on a location in which the app is rendered.
-  //
-  // NB: Location "app-config" is auto-included in the list as most apps need it
-  // You can remove it (and on the app definition also) in case the app
-  // doesn't require it
-  ComponentLocationSettings.forEach(componentLocationSetting => {
-    if (sdk.location.is(componentLocationSetting.location)) {
-      render(componentLocationSetting.component, root);
-    }
+    // Select a component depending on a location in which the app is rendered.
+    //
+    // NB: Location "app-config" is auto-included in the list as most apps need it
+    // You can remove it (and on the app definition also) in case the app
+    // doesn't require it
```

Comment:
How is it auto included? It seems to be just an element in the array like any other location